standalone/nginx/default.conf

```diff
@@ -17,18 +30,38 @@ server {
 
     location / {
         include /etc/nginx/snippets/proxy-params.conf;
+
         try_files $uri $uri/ /index.html;
         add_header Cache-Control no-cache;
     }
 
     location /api/ {
         include /etc/nginx/snippets/proxy-params.conf;
 
+        proxy_http_version 1.1;
+        proxy_set_header Connection "";
+
         # jmwalletd expects the bearer token in the Authorization header
         proxy_set_header Authorization $http_x_jm_authorization;
         # do not forward the custom authorization header
         proxy_set_header x-jm-authorization "";
 
-        proxy_pass https://127.0.0.1:28183;
+        proxy_pass https://jmwalletd_api_backend;
+    }
+
+    location = /jmws {
+        include /etc/nginx/snippets/proxy-params.conf;
+
+        proxy_http_version 1.1;
+
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection "upgrade";
+        proxy_set_header Authorization "";
+
+        # allow 24h without socket activity (default is 60 sec)
```

Comment:
24h seems feels a bit long to me compared to 60s. 😄

Reply:
What about 1h? Or even 10m? Do you have a number in mind?

We can even go for 60s now that disconnects are handled. User will see a short flash of the connection indicator if no wallet is unlocked, though.

Reply:
10m seems reasonable. We must cope with shorter times anyway.